tools/cpp/unix_cc_toolchain_config.bzl

```diff
@@ -1350,9 +1379,9 @@ def _impl(ctx):
             ),
         ]
         features = [
+            macos_minimum_os_feature,
             libtool_feature,
             archiver_flags_feature,
-            supports_pic_feature,
```

Comment:
This was not supposed to be here, but was just not getting exercised. see https://github.com/bazelbuild/bazel/issues/12439